src/types/global.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Window {
+    Countly: unknown;
+}
```

Comment:
this is untrue, and I don't think we want to put the Countly instance on the window object